src/replication.c

```diff
@@ -3132,13 +3130,13 @@ int replicaTryPartialResynchronization(connection *conn, int read_reply) {
         return PSYNC_TRY_LATER;
     }
 
-    if (!strncmp(reply, "+DUALCONNSYNC", strlen("+DUALCONNSYNC"))) {
-        /* A response of +DUALCONNSYNC from the primary implies that partial
+    if (!strncmp(reply, "+DUALCHANNSYNC", strlen("+DUALCHANNSYNC"))) {
```

Comment:
CHANN? A new abbreviation? Is the full word CHANNEL too long?

Reply:
Yeah, let's just use the full name here.